externs/opentype.js

```diff
@@ -1,651 +0,0 @@
-/**
```

Comment:
@yne this one's an important one.

Reply:
I removed it because:
- It is not documented
- It is not used/referenced by any config

I'll add it back but:
- Isn't it duplicate of existing minifier/builder/linter tools ?
- If it's better, why it is not used by default ?
- Why does it duplicate all opentype.js prototypes ? Can't we automatize it ?

Reply:
@yne btw, that benchmark i think has it running in simple optimizations mode, advanced optimizations mode is even better, but requires JSDoc documentation of all code OR an extern definition file for external libraries that are used (like opentype.js)

Reply:
Interesting, I would prefer to document the whole source (and enforce it in CI)
so we could avoid repeating public prototypes in /external/opentype.js

... but that's a subject for another PR ;)

Reply:
I just run `eslint` with `jsdoc` enforcement, and I got 1819 errors ... (missing type, returns, description ...) lot of work to do ...

Reply:
@yne just a note, closure compiler uses a specific sub-set of jsdoc syntax, so you might want to look at their documentation if you're modifying the /externs/opentype.js file. Closure compiler basically requires externs for all libraries that are loaded separately so it knows what it can and can't optimize. Also, personally i think the best solution would be to annotate all public APIs as such in JSDoc using the  ``@public`` and ``@private`` annotations, and then generate the extern stubs programmatically based on those JSDocs when we tag a release (i.e. create a stub for everything that is marked with ``@public``), but that should probably be a separate PR. I can help with that one as I have some experience with closure compiler.

Reply:
That would be a very nice improvement.

We could also automatically generate the API documentation and examples of any `@public` methods.